factcast-store-pgsql/src/main/java/org/factcast/store/pgsql/registry/transformation/TransformationConfiguration.java

```diff
@@ -41,6 +44,16 @@ public TransformationStore transformationStore(@NonNull JdbcTemplate jdbcTemplat
         return new InMemTransformationStoreImpl();
     }
 
+    @Bean
+    public TransformationCache transformationCache(@NonNull JdbcTemplate jdbcTemplate,
+            @NonNull PgConfigurationProperties props, @NonNull SpringLiquibase unused) {
```

Comment:
needs to be @Autowired(required=false) for liquibase

Reply:
Changed it in both places